src/actions/index.js

```diff
@@ -11,8 +11,14 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+ return{
+   type: INCREMENT
+ }
```

Comment:
Good job on this. You could have added a payload, but you recognized that you didn't need one. 👍